ini.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of the MAVLink Router project
+ *
+ * Copyright (C) 2017  Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+class IniFile {
+public:
+    IniFile(const char *_filename)
+        : filename{_filename}
+        , sections{nullptr}
+        , current_section{nullptr} {};
```

Comment:
no. This is:

```
class Bla {
    int bli = 42;
    const char *bla = "42";
};
```

Use the initialization list only for stuff that is going to change when the arguments change